### PR TITLE
Add hover help text

### PR DESCRIPTION
**Pull request by Ozrei**

### CODE REVIEW

File: keyform/forms.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super(CreateForm, self).__init__(*args, **kwargs)
         # removes blank choices from Radio Select options
-        self.fields['payment_method'] = TypedChoiceField(widget=RadioSelect(), choices=Request.PAYMENT_TYPES)
+        self.fields['payment_method'] = TypedChoiceField(widget=RadioSelect(), choices=Request.PAYMENT_TYPES, help_text="Cash/Check should only be accepted during camps and conferences, and also fill in the amount received. Use the Charge Amount box to change to the student's account, or mark that the student was charged on the RCR if they are checking out.")
```
Comment (enki211) [inline keyform/forms.py]: Translation also this looks like the line is too long.

File: keyform/models.py [hunk 2]
```diff
@@ -49,16 +49,16 @@ class Request(models.Model):
...
-    building = models.ForeignKey(Building, help_text='')
-    student_name = models.CharField(max_length=128, blank=True, help_text='')
-    reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES, help_text='')
-    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name= _('Amount received'), validators=[MinValueValidator(Decimal('0.00'))], help_text='')
-    payment_method = models.CharField(max_length=2, choices=PAYMENT_TYPES, help_text='')
-    charge_amount = models.DecimalField(max_digits=7, default=0, decimal_places=2, validators=[MinValueValidator(Decimal('0.00'))], help_text='')
+    building = models.ForeignKey(Building)
+    student_name = models.CharField(max_length=128, blank=True)
+    reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES)
+    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name= _('Amount received'), validators=[MinValueValidator(Decimal('0.00'))], help_text='Core Change/Reprogram Fob Charge = $50, Room Key/Fob = $10, Mailbox Key = $10, Monroe Mailbox Key = $25')
```
Comment (enki211) [inline keyform/models.py]: Translation, and is the line too long?

File: keyform/models.py [hunk 3]
```diff
@@ -80,11 +80,11 @@ class KeyData(models.Model):
     )
 
     request = models.ForeignKey(Request)
-    core_number = models.CharField(max_length=35, verbose_name=_('New Core Number'), blank=True, help_text='')
-    key_type = models.CharField(max_length=2, choices=KEY_TYPES, help_text='')
-    room_number = models.CharField(max_length=42, help_text='')
-    key_number = models.CharField(max_length=24, verbose_name=_('Lost/Stolen/Damaged Key Number'), help_text='')
-    quantity = models.IntegerField(validators=[MinValueValidator(0)],help_text='')
+    core_number = models.CharField(max_length=35, verbose_name=_('New Core Number'), blank=True, help_text='Use a comma to separate the list of cores (if suite style).')
```
Comment (enki211) [inline keyform/models.py]: Translation, and is the line too long?

File: keyform/models.py [hunk 4]
```diff
@@ -80,11 +80,11 @@ class KeyData(models.Model):
...
+    key_type = models.CharField(max_length=2, choices=KEY_TYPES)
+    room_number = models.CharField(max_length=42)
+    key_number = models.CharField(max_length=24, verbose_name=_('Lost/Stolen/Damaged Key Number'))
+    quantity = models.IntegerField(validators=[MinValueValidator(0)], help_text="If you don't need to order more keys because there are enough already, you can mark zero.")
```
Comment (enki211) [inline keyform/models.py]: Translation, and is the line too long?

**Comment by shaneallgeier** [comment]
Why did you delete the help-text spans from everywhere?

**Comment by Ozrei** [comment]
Clearly, if there are still help-text spans, I didn't delete them from "everywhere"

**Comment by shaneallgeier** [comment]
Why did you delete **most of** the help-text spans from **almost** everywhere?

**Comment by Ozrei** [comment]
Because Teresa/Angie only gave us help text for certain areas, and it would've been superfluous to just have them sitting around, empty and useless. ~~Like your heart.~~

File: keyform/migrations/0010_auto_20170811_1848.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        ),
+        migrations.AlterField(
+            model_name='request',
+            name='amt_received',
+            field=models.DecimalField(blank=True, decimal_places=2, default=0, help_text='Core Change/Reprogram Fob Charge = $50, Room Key/Fob = $10, Mailbox Key = $10, Monroe Mailbox Key = $25', max_digits=7, validators=[django.core.validators.MinValueValidator(Decimal('0.00'))], verbose_name='Amount received'),
+        ),
+        migrations.AlterField(
+            model_name='request',
+            name='payment_method',
+            field=models.CharField(choices=[('ca', 'Cash'), ('ch', 'Check'), ('na', 'Not Applicable')], help_text="Cash/Check should only be accepted during camps and conferences, and also fill in the amount received. Use the Charge Amount box to change to the student's account, or mark that the student was charged on the RCR if they are checking out.", max_length=2),
```
Comment (shaneallgeier) [inline keyform/migrations/0010_auto_20170811_1848.py]: You'll need to create a new migration since this help_text is no longer part of models.py

**Comment by Patrick7321** [comment]
Listen, I like this code. 'Tis good. But here is the problem. Users can be dumb. They are like 'wut woah, how to do the hover help text thingy mcbobber?' So I think you should provide some extra indicators so even the most ridiculous user can find victory.

File: keyform/static/keyform/css/add_form.css [hunk 6]
```diff
@@ -21,3 +21,11 @@ div.row {
 .keydata-form:last-of-type .generate_forms {
     visibility: visible;
 }
+
+.help-text::after {
+	content: "\2754";
```
Comment (shaneallgeier) [inline keyform/static/keyform/css/add_form.css]: You used a tab character instead of 4 spaces

File: keyform/static/keyform/css/add_form.css [hunk 7]
```diff
@@ -21,3 +21,11 @@ div.row {
...
+}
+
+.help-text label {
+	display: inherit;
```
Comment (shaneallgeier) [inline keyform/static/keyform/css/add_form.css]: You used a tab character instead of 4 spaces

**Pull request merged**